res/database/Default/database_people.xml

```diff
@@ -1981,1013 +2456,1014 @@
-		<person id="a28a8839-5001-4a60-bbd8-ea590e141724" first_name="Maurice Dean" last_name="Wintson" nick_name="" />
-		<person id="fbb31784-0a91-4975-b240-ab0aff65f052" first_name="Joseph" last_name="Wisemenn" nick_name="" imdb_id="nm0936476" />
-		<person id="65038a76-0d9b-43a2-a19c-92818a256c0f" first_name="Robert" last_name="Wisesan" nick_name="" imdb_id="nm0936404" />
-		<person id="1f36e89d-8aaa-436e-8664-df837a2678dc" first_name="Ray" last_name="Wiseson" nick_name="" imdb_id="nm0936403" />
-		<person id="ad21cfc7-d8b0-4b10-b49d-ec3822b4a0f1" first_name="Michael C." last_name="Wjelaiams" nick_name="" imdb_id="nm0931321" />
-		<person id="9e19b198-4367-4637-ae6b-bb43264f8b4a" first_name="Walther Röhrli und Jack" last_name="Wjellsniff." nick_name="" />
-		<person id="63d730f3-c0a5-4ab0-97f7-250177c5f3bd" first_name="Joey" last_name="Wongrol" nick_name="" />
-		<person id="207383e7-cfef-43ee-97be-4af1dc62520b" first_name="John" last_name="Woodson" nick_name="" imdb_id="nm0939795" />
-		<person id="6cb07810-f86a-4dd4-bf9a-3b2afb574826" first_name="Natalie" last_name="Woodson" nick_name="" imdb_id="nm0000081" />
-		<person id="a61f1eba-e305-404c-833d-3932c445bdc1" first_name="Yames" last_name="Woulds" nick_name="" imdb_id="nm0000249" />
-		<person id="d1eb3b10-594b-4e3c-b74e-96e5f85c1d6d" first_name="Kris" last_name="Wredges" nick_name="" imdb_id="nm0917188" />
-		<person id="888ecdf4-4e26-4ce2-9c49-e86637aefb9d" first_name="Hank" last_name="Wulf" nick_name="" imdb_id="nm0938231" />
-		<person id="61334348-7cb7-4601-ad29-87f724ec3cb3" first_name="Ma" last_name="Wurol" nick_name="" />
-		<person id="e93341f1-9085-49b5-834b-a97d3d6a85a4" first_name="Geraint" last_name="Wyn Tavies" nick_name="" />
-		<person id="c432964f-a50a-49a1-8e75-fd448ec2530b" first_name="Taro" last_name="Yamhamoto" nick_name="" imdb_id="nm0945500" />
-		<person id="1d6ee92b-0095-4a98-80ca-e82114788fe4" first_name="Rudy" last_name="Youngbloodrol" nick_name="" imdb_id="nm2199632" />
-		<person id="d74284c8-2694-4f9e-812a-e57aa0c2a227" first_name="Terence" last_name="Youngson" nick_name="" imdb_id="nm0950109" />
-		<person id="0142e7e1-c601-4de6-ab42-f3a34bc3c2dd" first_name="Karl-Peter" last_name="Ziflecker" nick_name="" />
-		<person id="bba20871-8558-47e7-948d-af5476e0d789" first_name="Tsang" last_name="Zimao" nick_name="" imdb_id="nm0955443" />
-		<person id="3bfe0f5c-eb37-4559-9300-4fe0ab3bdb9b" first_name="Fred" last_name="Zinmeemahn" nick_name="" imdb_id="nm0003593" />
-		<person id="3535a46a-f9e9-4519-a5de-2a2566b0adf1" first_name="Wim" last_name="Zomerer" nick_name="" imdb_id="nm0957779" />
-		<person id="fee7aeab-0e92-4cbf-82e5-6e456583fa51" first_name="Edward" last_name="Zwigg" nick_name="" imdb_id="nm0001880" />
+		<person id="60318881-ed5f-4b1e-89cf-f950940d4424" first_name="Reynaud" last_name="Goscogne" nick_name="" first_name_original="René" last_name_original="Goscinny" birthday="1926-08-14" deathday="1977-11-05" imdb_id="nm0331453" />
```

Comment:
Github does not show the diff well for about 600 lines (one-line changes), has to be checked in IDE